server/1.2/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM fabric8/java-centos-openjdk11-jdk
+
+LABEL maintainer="Debezium Community"
+
+#
+# Set the version, home directory, and MD5 hash.
+#
+ENV DEBEZIUM_VERSION="1.2.1.Final" \
+    SERVER_HOME=/debezium \
+    SERVER_MD5=3f39de479ba7fc90316d127aa9b5f7d5
+ENV SERVER_URL_PATH=io/debezium/debezium-server-dist/$DEBEZIUM_VERSION/debezium-server-dist-$DEBEZIUM_VERSION.tar.gz
+
+#
+# Create a user and home directory for Debezium Server
+#
+USER root
+RUN groupadd -r debezium -g 1001 && \
+    useradd -u 1001 -r -g debezium -m -d $SERVER_HOME -s /sbin/nologin -c "Debezium Server user" debezium && \
+    chmod 755 $SERVER_HOME
+
+#
+# Change ownership and switch user
+#
+RUN chown -R debezium $SERVER_HOME && \
+    chgrp -R debezium $SERVER_HOME
+USER debezium
+
+RUN mkdir $SERVER_HOME/conf && \
```

Comment:
wouldn't `config` be more verbose and match the other images structure? (like `/kafka/config/` in connect-base / connect image)

Reply:
This is aligned with Debezium Server default naming.